packages/inter-protocol/src/auction/scheduleMath.js

```diff
@@ -147,3 +147,27 @@ export const nextDescendingStepTime = (liveSchedule, nextSchedule, now) => {
   return expectedNext;
 };
 harden(nextDescendingStepTime);
+
+/**
+ *
+ * @param {Timestamp} time
+ * @param {import('./scheduler.js').Schedule} schedule
+ * @returns {'before' | 'during' | 'endExactly' | 'after'}
+ */
+export const timeVsSchedule = (time, schedule) => {
```

Comment:
Is this supposed to read "time versus schedule"? If not, I don't understand the name.

Reply:
yes